bitbots_head_mover/src/head_capsule.cpp

```diff
@@ -0,0 +1,275 @@
+#include <cmath>
+#include <rclcpp/rclcpp.hpp>
+#include "tf2_ros/transform_listener.h"
+#include "tf2_ros/buffer.h"
+#include <tf2_ros/transform_broadcaster.h>
+
+#include <humanoid_league_msgs/msg/head_mode.hpp>
+#include <bitbots_msgs/msg/joint_command.hpp>
+#include <sensor_msgs/msg/joint_state.hpp>
+// #include <bitbots_moveit_bindings/check_collision.hpp> use moveit directly
+
+class HeadCapsule : public rclcpp::Node
+{
+public:
+  HeadCapsule()
+      : Node("head_capsule") // maybe use a node given as parameter and not inherit from node
```

Comment:
I am not sure if we would want to keep the head capsule separated if we do not use the dsd.